### PR TITLE
[DSS] DDP-8151: Move import of BrowserAnimationsModule from ddp-sdk to every single study

### PR DESCRIPTION
**Pull request by alextanlan**
The issue is described [in DDP-8151 ticket](https://broadinstitute.atlassian.net/browse/DDP-8151).

### CODE REVIEW

File: ddp-workspace/projects/ddp-sdk/src/lib/ddp.module.ts [hunk 1]
```diff
@@ -238,7 +237,6 @@ export function createTranslateLoader(http: HttpClient): TranslateHttpLoader {
 @NgModule({
     imports: [
         CommonModule,
-        BrowserAnimationsModule,
```
Comment (alextanlan) [inline ddp-workspace/projects/ddp-sdk/src/lib/ddp.module.ts:241]: Removed from shared `ddp-sdk` module. 
Move import of `BrowserAnimationsModule` to `app.module` of every single study.

File: ddp-workspace/projects/ddp-singular/src/app/app.module.ts [hunk 2]
```diff
@@ -1,6 +1,7 @@
-import { AnalyticsEventCategories, AnalyticsEventsService, DdpModule } from 'ddp-sdk';
```
Comment (alextanlan) [inline ddp-workspace/projects/ddp-singular/src/app/app.module.ts:1]: Removed unused import of `AnalyticsEventCategories`.

**Review by GiCharkviani — APPROVED** [review]

**Review by josemanuel167 — APPROVED** [review]

**Comment by MocanaAtBroad** [comment]
If we missed updating one of the Angular study projects to import the module: would it cause a compilation problem? Or is it a runtime error?

**Comment by alextanlan** [comment]
@MocanaAtBroad 
Unfortunately, it is a runtime error.

**Review by MocanaAtBroad — APPROVED** [review]

**Pull request merged**